common/datavalues/src/columns/column.rs

```diff
@@ -192,6 +197,51 @@ where A: AsRef<dyn Array>
             }),
         ))
     }
+
+    fn into_column_with_field(self, f: &ArrowField) -> ColumnRef {
+        use TypeID::*;
+        match from_arrow_field_meta(f) {
+            Some(data_type) => {
+                match data_type.data_type_id() {
+                    // arrow type has no nullable type
+                    Nullable => unimplemented!(),
```

Comment:
Maybe Err(ErrorCode::LogicaError) is better ?

Reply:
I think `unimplemented` is ok, this is not a logical error, nullable is defined by ourselves, `arrow` doesn't have a corresponding type.
what is your opinion? @sundy-li

Reply:
`unimplemented!()` will panic, please use the error instead.

Reply:
We can use `ErrorCode::UnImplement` instead.

Reply:
ok, we can modify it in other issues, as the implementation of nullable is not related to varint